script/lgbtq.yml

```diff
@@ -0,0 +1,152 @@
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - gay
+    - gay man
+    - lesbian
+    - gay person/people
+  inconsiderate:
+    - homosexual
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - relationship
+  inconsiderate:
+    - homosexual relations
+    - homosexual relationship
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - couple
+  inconsiderate:
+    - homosexual couple
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - sexual orientation
+    - orientation
+  inconsiderate:
+    - sexual preference
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - gay lives
+    - gay/lesbian lives
+  inconsiderate:
+    - gay lifestyle
+    - homosexual lifestyle
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - sexual orientation
+    - orientation
+  inconsiderate:
+    - sexual preference
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - gay issues
+  inconsiderate:
+    - gay agenda
+    - homosexual agenda
+- type: simple
+  source: http://www.glaad.org/reference/style
+  considerate:
+    - equal rights
+    - civil rights for gay people
+  inconsiderate:
+    - special rights
+    - gay rights
+- type: simple
+  source: http://www.glaad.org/reference/offensive
+  considerate:
+    - gay
+  inconsiderate:
+    - fag
+    - faggot
+    - dyke
+    - homo
+    - sodomite
+- type: simple
+  source: http://www.glaad.org/reference/style
+  considerate:
+    - bisexual
+  inconsiderate:
+    - bi
+  - type: simple
+  source: http://www.glaad.org/reference/style
+  considerate:
+    - gay marriage
+    - same-sex marriage
+  inconsiderate:
+    - homosexual marriage
+- type: simple
+  source: http://www.glaad.org/reference/style
+  considerate:
+    - transgender
+  inconsiderate:
+    - tranny
+- type: simple
+  source: http://www.glaad.org/reference/transgender
+  considerate:
+    - cross-dresser
+  inconsiderate:
+    - transvestite
+- type: simple
+  source: http://www.glaad.org/reference/transgender
+  considerate:
+    - transition
+  inconsiderate:
+    - sex-change
```

Comment:
Could you provide examples of where theses cases might be considered insensitive?